{"name": "test_sample_ui_test_case", "status": "broken", "statusDetails": {"message": "AttributeError: 'WebDriver' object has no attribute 'driver'", "trace": "self = <TestCases.testsuites.test_sample.TestSample1 object at 0x000001FA2B23BB10>\nuiTestData = {'discharge_date': '01/18/2023', 'facility': {'facility_id': 'FA03', 'facility_name': 'Teamfacility'}, 'member': {'DOB...TN23', 'Member_id': 'TN22', ...}, 'password': '80f7d73c5456-430da38f0b7de6131f170c57755653164ab2a353a1cc138d12b9', ...}\nconfigLogger = <RootLogger root (INFO)>\n\n    def test_sample_ui_test_case(self,uiTestData, configLogger):\n        driver = webdriver.Chrome(service=Service(ChromeDriverManager().install()))\n        driver.maximize_window()\n        login(uiTestData=uiTestData,driver=driver)\n>       login_web(uiTestData=uiTestData,driver=driver)\n\nTestCases\\testsuites\\test_sample.py:35: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nCommonClass\\web_utils\\CommonUiActions.py:44: in login_web\n    Member.createMember(driver)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"8dcf1c0cb44469514b0db01a9824ccaa\")>\n\n    def createMember(self):\n>       self.driver.find_element(By.XPATH, self.member_search_id).click()\nE       AttributeError: 'WebDriver' object has no attribute 'driver'\n\nCommonClass\\web_utils\\pages\\member.py:15: AttributeError"}, "steps": [{"name": "Log (INFO) ====== WebDriver manager ======", "status": "passed", "start": 1674127439967, "stop": 1674127439967}, {"name": "Log (INFO) Get LATEST chromedriver version for google-chrome 108.0.5359", "status": "passed", "start": 1674127442886, "stop": 1674127442886}, {"name": "Log (INFO) Driver [D:\\Users\\pvelu\\.wdm\\drivers\\chromedriver\\win32\\108.0.5359\\chromedriver.exe] found in cache", "status": "passed", "start": 1674127445198, "stop": 1674127445198}], "attachments": [{"name": "log", "source": "2c97658b-b384-4d71-a108-07c055777a91-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "1406d8f2-9db9-4c2f-b5e9-a3270eb4c0ce-attachment.txt", "type": "text/plain"}], "start": 1674127439966, "stop": 1674127468688, "uuid": "9bb3c79f-96bb-4ded-98d9-819b4e3d0497", "historyId": "1971dd38761290445127c4af03b141fd", "testCaseId": "2186558a34288d22382cd9d3e416f696", "fullName": "TestCases.testsuites.test_sample.TestSample1#test_sample_ui_test_case", "labels": [{"name": "parentSuite", "value": "TestCases.testsuites"}, {"name": "suite", "value": "test_sample"}, {"name": "subSuite", "value": "TestSample1"}, {"name": "host", "value": "WSAMZN-7K50OVEF"}, {"name": "thread", "value": "6384-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.testsuites.test_sample"}]}